packages/components/src/avatar/src/Avatar.css

```diff
@@ -1,58 +1,51 @@
 .o-ui-avatar {
     display: inline-flex;
     flex-shrink: 0;
-    border-radius: var(--o-ui-br-circular);
+    border-radius: var(--hop-shape-circle);
     overflow: hidden;
 }
 
 /* SIZES */
-.o-ui-avatar-2xs {
-    width: var(--o-ui-sz-3);
-    height: var(--o-ui-sz-3);
+.o-ui-avatar-xs,
+.o-ui-avatar-sm,
+.o-ui-avatar-md,
+.o-ui-avatar-lg,
+.o-ui-avatar-xl,
+.o-ui-avatar-image {
+    aspect-ratio: 1/1;
```

Comment:
can't this be directly in o-ui-avatar? is there some sizes where the aspect ratio must not be set?

Reply:
good point I'll make the change